### ISSUE TITLE
Get surface geopotential from the grid manager rather than passing it through the field manager

### ISSUE THREAD

**Issue by PeterCaldwell** [E3SM-Project/scream#958]
Currently some processes ask the field manager for PHIS. But PHIS is a time-invariant property of the model grid, so makes better logical sense to be supplied by the grid manager. Tasks:

- [ ] Make a function associated with the grid which returns surface geopotential
- [ ] Modify processes asking for PHIS to instead use this feature. It seems like PHIS should be defined during initialization time then held in memory forever afterwards? Though maybe it would be more memory efficient to just recalculate it every timestep?
- [ ] quit calling it PHIS, which only makes sense to E3SM afficionados. Call it surf_geopot or something instead...

**Comment by bartgol** [comment]
How would one recompute phis? `phi=-gz`, so phis is basically the topography, right? If you want to recompute it, you need to store the topography in the grid. One of the two must be available. I'm ok with storing surface elevation, instead of surface geopotential (it might be even clearer). But one of the two needs to be in the grid.

**Comment by PeterCaldwell** [comment]
Yeah, I like storing surface height rather than surface geopotential. And  like you say, I don't envision "recomputing" geopotential because it is really just something that needs to be read in from a file. I was just imagining that the grid would include surface elevation data and would have a method to just return that data when asked. Though I guess if grid is an object, we could just as easily ask  for grid.surface_height as asking for surface_height = grid.get_surface_height() - which is what I was originally thinking of above.

**Comment by bartgol** [comment]
Well, that assumes that we have a `surface_height` member of the base abstract class. We already store "geometry data" views, in a `std::map<std::string, view1d_type>`. So we don't even have to change the base class interface. We simply need to make sure our grids store such a view.

**Comment by AaronDonahue** [comment]
@bartgol or @tcclevenger , based on the conversation in #1313 is this still even a relevant issue?  Can we close it?

**Comment by bartgol** [comment]
I think we can close, yes. The topography can def stay in the FM.
